designsafe/apps/notifications/views.py

```diff
@@ -96,7 +96,7 @@ def generic_webhook_handler(request):
     except (HTTPError, AgaveException, ApiException) as e:
         LOGGER.exception(
             "Could not find valid corresponding Agave job for interactive event")
-        return HttpResponse(json.dumps(e.message), content_type='application/json', status=400)
+        return HttpResponse(json.dumps(e), content_type='application/json', status=400)
```

Comment:
same here and on line 117. an unrelated bug to what you were originally fixing in this pr.

Reply:
good catch. will fix in a new PR.